save-demo/src/main/kotlin/com/saveourtool/save/demo/runners/Runner.kt

```diff
@@ -0,0 +1,37 @@
+package com.saveourtool.save.demo.runners
+
+import com.saveourtool.save.demo.DemoAdditionalParams
+import com.saveourtool.save.demo.DemoResult
+import java.nio.file.Path
+
+/**
+ * Interface should be implemented by all the runners.
+ */
+interface Runner<in P : DemoAdditionalParams, out R : DemoResult> {
+    /**
+     * Save [lines] into file with [filePath]
+     *
+     * @param filePath path to a new file
+     * @param lines string that should be printed to file, if null, nothing is done
+     * @return file with [lines]
+     */
+    fun prepareFile(filePath: Path, lines: String?) = lines?.let { fileLines ->
+        filePath.toFile().apply {
```

Comment:
Path also has `writeText`: `import kotlin.io.path.writeTextimport kotlin.io.path.writeText`

Reply:
Oh, will replace it then